esmvalcore/_recipe.py

```diff
@@ -845,6 +899,16 @@ def append(group_prefix, var):
     return derive_input
 
 
+def _remove_time_preproc(fxprofile):
```

Comment:
It's a bit confusing that this function doesn't modify in place. Maybe a name like `_get_filtered_fxprofile` would be better?

Reply:
done :beer: